module-4/backend/week_one.md

```diff
@@ -4,17 +4,56 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. What's the most useful thing you learned from completing the intermission week work?
-2. What are some tools to help debug JavaScript code?
-3. What are some tools you need in order to unit test your JavaScript?
-4. What is the syntax for invoking a function?
-5. What's the difference between `==` and `===` in JavaScript?
-6. What's the difference between asynchronous and synchronous JavaScript? 
-7. How do we setup a route when creating an API with Node and Express?
-8. What do we use Knex for?
-9. What's `npm` and what do we use it for?
+1. What's the most useful thing you learned from completing the intermission week work?  
+
+   Becoming comfortable with basic JavaScript.  
+   
+   
+2. What are some tools to help debug JavaScript code?  
+
+   debugger, chrome Dev Tools, pryJs, and my old favorite: console.log, seriously though, after Friday's lesson I've primarily used Chrome Dev Tools   
+   
+   
+3. What are some tools you need in order to unit test your JavaScript?  
+
+   mocha, chai  
+   
+   
+4. What is the syntax for invoking a function?  
+
+   functionName()  
+   
+   
+5. What's the difference between `==` and `===` in JavaScript?  
+
+   `==` is more strict than `===`  
```

Comment:
These questions (5,6, 7,8) are from an outdated version of this repo. Please be sure to update your repo for next week.

`===` (called strict equal) is actually more strict than `==`. It first ensures that the objects are of the same type before performing an equal while `==` coerses one the the values to be the same type as the other